gpu/amd_linux.go

```diff
@@ -41,7 +41,7 @@ const (
 
 var (
 	// Used to validate if the given ROCm lib is usable
-	ROCmLibGlobs          = []string{"libhipblas.so.2*", "rocblas"} // TODO - probably include more coverage of files here...
+	ROCmLibGlobs          = []string{"libhipblas.so*", "rocblas"} // TODO - probably include more coverage of files here...
```

Comment:
I think this will be problematic.  For our official builds linked against ROCm v6, this will allow us to detect and accept a ROCm v5 which wont work at runtime.  Perhaps the simplest change to accommodate both use cases is to have a simple string variable that can be overridden.
```go
var (
    ROCmLib = "libhipblas.so.2*"
    ROCmLibGlobs = []string{ROCmLib, "rocblas"}
)
```

Then at buildtime when linking against v5, 
```
export GOFLAGS="'-ldflags=-w -s \"-X=github.com/ollama/ollama/gpu.ROCmLib=libhipblas.so.1*\"'"
```

Reply:
This would be fine for us as well.

Reply:
I have now pushed the changes you suggested. Can you have another look?